### ISSUE TITLE
Select Individual folders

### ISSUE THREAD

**Issue by mizu-048** [DineshSolanki/FoliCon#222]
**Describe the solution you'd like**
Currently, you can only select one folder, whose subfolder are selected for cover update, excluding the select folder itself
**Additional context**
and how do you reselect a folder present among many folder to update it covers
like making icons for the same folder skips already updated folders

**Comment by DineshSolanki** [comment]
You can press skip button for folders you don't want to process, currently that's the only way.

**Comment by mizu-048** [comment]
oh this is meant to be feature request

**Comment by DineshSolanki** [comment]
This case is covered by skip feature so. If you add more context on situation, will surely look into the details

**Comment by mizu-048** [comment]
just because a case is covered doesn't mean its the best or even a good way to do it
you would have to press skip for every folder, manually and before that you press cancel for every icon search(in professional mode) about which i might make another issue someday since its runs infinitely)

**Comment by DineshSolanki** [comment]
@mizu-048 do you have idea in your mind like how would you like it to be, like a check box?

**Comment by mizu-048** [comment]
since there are many options to consider, i think it would be best if after one or more folders are select, the user is prompted with various checkboxes with options like
- include subfolders
- include folders which already have an icon
maybe a subfolder input box as well to use for only this search

although not to complicate stuff, the ability to just select more than 1 folders, which are chosen for icon update
and do a notif popup if some folders already have an icon warning that they will be overwrriten or if we want to skip

**Comment by DineshSolanki** [comment]
@mizu-048 agreed on the complexity, I'm thinking about the design as to not clutter user with too many popuos buttons etc.

Will think in this, suggestions and ideas from anyone are appreciated.

**Comment by HekutoruMAC** [comment]
Dealing with unwanted children folders could be achieved by adding a stop button to download previously chosen icons.
Even better, handle it with a maximum/intended folder depth level selector instead of the endless subfolder processing checkbox.

**Comment by DineshSolanki** [comment]
> Dealing with unwanted children folders could be achieved by adding a stop button to download previously chosen icons.
> Even better, handle it with a maximum/intended folder depth level selector instead of the endless subfolder processing checkbox.

I liked the idea of choosing the depth of folder, will put that in my list

**Comment by DineshSolanki** [comment]
Hi @HekutoruMAC , @mizu-048 this feature is now available.
